stepfunctions-lambda/README.md

```diff
@@ -0,0 +1,43 @@
+# Creating a Lambda function with a function URL
+
+In this example, we will demonstrate how to create an AWS Step Function with Lambda functions in LocalStack.
+
+## Prerequisites
+
+* LocalStack
+* Docker
+* `awslocal` CLI
+
+## Starting up
+
+Start LocalStack via: 
+
+```sh
+localstack start -d
+```
+
+Push the following command to create the Lambda functions:
+
+```sh
+make create-lambdas
+```
+
+## Setting up and running Step Function
+
+Create the Step Function:
+
+```sh
+awslocal stepfunctions create-state-machine --name step-demo \
+  --definition "$(cat step-definition.json)" \
```

Comment:
Why is this not using the same pattern as above in the Makefile? (`file://....`)

Reply:
Its not needed here